exercises/concept/csv-builder/.meta/exemplar.rs

```diff
@@ -30,3 +30,9 @@ impl CsvRecordBuilder {
         self.content
     }
 }
+
+impl Default for CsvRecordBuilder {
+    fn default() -> Self {
+        Self::new()
+    }
+}
```

Comment:
A type with a `fn new() -> Self` function should also have a `Default` implementation.